### PR TITLE
Improve Groovy Grapes configuration:

### PR DESCRIPTION
**Pull request by jdrossl**
- Ignore the local m2 repo & external cached grapes to avoid conflicts
- Make Tomcat use a custom Grapes config & folder
- Remove some windows specific code

https://github.com/craftercms/craftercms/issues/3326

### CODE REVIEW

File: resources/bin/grapeConfig.xml [hunk 1]
```diff
@@ -2,12 +2,8 @@
 <ivysettings>
   <settings defaultResolver="downloadGrapes"/>
   <resolvers>
-    <chain name="downloadGrapes">
-      <!-- no local m2 repo is defined to avoid conflicst with depedencies in m2 -->
-      <filesystem name="cachedGrapes">
-        <ivy pattern="${user.home}/.groovy/grapes/[organisation]/[module]/ivy-[revision].xml"/>
```
Comment (avasquez614) [inline resources/bin/grapeConfig.xml]: Why was this removed? Is it not necessary to define were grapes stores the cached deps? What happens if Authoring or Delivery don't have outside access to download the Grapes?

Comment (avasquez614) [reply resources/bin/grapeConfig.xml]: Ok I think I get it, it's just not using the ones in the user's home folder. Have you tried testing if there are any conflicts with the pre-cached grapes that the Groovy scripts like init site pull when rendering a site that also pulls Grapes?

Comment (jdrossl) [reply resources/bin/grapeConfig.xml]: I did a simple test adding a dependency to a site and both seem to work fine
